File: reactor-java/src/main/java/com/github/mostroverkhov/r2/reactor/RequesterBuilder.java
package com.github.mostroverkhov.r2.reactor;

import com.github.mostroverkhov.r2.core.CoreRequesterBuilder;
import com.github.mostroverkhov.r2.reactor.adapters.RequesterAdapter;
import io.rsocket.RSocket;

public class RequesterBuilder extends CoreRequesterBuilder {
    public RequesterBuilder(RSocket rSocket) {
        adapter(new RequesterAdapter(rSocket));
    }
}

File: rxjava-kotlin/src/main/java/com/github/mostroverkhov/r2/rxjava/internal/RequesterBuilder.kt
package com.github.mostroverkhov.r2.rxjava.internal

import com.github.mostroverkhov.r2.rxjava.internal.adapters.RequesterAdapter
import com.github.mostroverkhov.r2.core.CoreRequesterBuilder
import io.rsocket.kotlin.RSocket

internal class RequesterBuilder(rSocket: RSocket) : CoreRequesterBuilder() {
    init {
        adapter(RequesterAdapter(rSocket))
    }
}